app/master/cluster_master.py

```diff
@@ -70,12 +70,13 @@ def start_heartbeat_tracker_thread(self):
         Thread(target=self._start_heartbeat_tracker, name='HeartbeatTrackerThread', daemon=True).start()
 
     def _start_heartbeat_tracker(self):
-        self._hb_scheduler.enter(0, 0, self._disconnect_non_heartbeating_slaves)
+        self._hb_scheduler.enter(self._unresponsive_slaves_cleanup_interval, 0,
```

Comment:
Just call `_disconnect_non_heartbeating_slaves()` to init the heartbeat tests

Reply:
Oh I see. Makes sense, but we want to wait for `_unresponsive_slaves_cleanup_interval` seconds after the master starts up. That way heartbeat doesn't run before the slaves get a chance to connect. Is there another way to achieve that?

Reply:
Why wait? I don't think its worth the extra complexity in the code.